TenancyInformationApi/V1/Factories/TenancyResponseFactory.cs

```diff
@@ -0,0 +1,29 @@
+using System.Globalization;
+using TenancyInformationApi.V1.Boundary.Response;
+using TenancyInformationApi.V1.Domain;
+
+namespace TenancyInformationApi.V1.Factories
+{
+    public static class TenancyResponseFactory
+    {
+        public static TenancyInformationResponse ToResponse(this Tenancy tenancy)
+        {
+            return new TenancyInformationResponse
+            {
+                TenancyReference = tenancy?.TenancyReference,
+                CommencementOfTenancyDate = tenancy?.CommencementOfTenancyDate.ToString(CultureInfo.CurrentCulture),
```

Comment:
What if you try `ToString("O")`

Reply:
Good catch, I forgot to change this to match the swaggerdoc.
These dates should use `ToString("yyyy-MM-dd")` in that case.

```suggestion
                CommencementOfTenancyDate = tenancy?.CommencementOfTenancyDate.ToString("yyyy-MM-dd"),
```